src/app/onboarding/components/create-new-archive/create-new-archive.component.scss

```diff
@@ -0,0 +1,124 @@
+@import '../welcome-screen/welcome-screen.component.scss';
+
+.two-buttons {
+  button {
+    @media only screen and (max-width: 950px) {
```

Comment:
I believe SASS supports developer-defined [variables](https://sass-lang.com/documentation/variables); `950px` seems like a good candidate for such a variable, as is maybe this entire `@media only screen and (max-width: 950px)`, since we use it half a dozen times in this file.